spec/system/task_spec.rb

```diff
@@ -1,13 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'タスク管理機能', type: :system do
+  before do
+    @task = create(:task)
+    @task = create(:second_task)
```

Comment:
5行目で@taskに作成を行なった後、6行目で@taskに再度代入を行なっていますが、
`@task, @second_task`と分けた方が良いのではないでしょうか？

また、この部分はletで記載することも可能です。
```suggestion
    let(:task) { create(:task) }
    let(:second_task) { create(:second_task) }
```

Reply:
ご指摘ありがとうございます。テストを通過していたので見落としていました。こちら修正いたします。また、letを活用した記載も今後の参考にさせていただきます。